src/sentry/runner/initializer.py

```diff
@@ -187,6 +188,15 @@ def configure_structlog():
 
     structlog.configure(**kwargs)
 
+    lvl = os.environ.get('SENTRY_LOG_LEVEL', None)
+    if lvl in logging._levelNames:
+        from sentry.conf.server import LOGGING as base_dict
```

Comment:
We'd want to pull `from django.conf import settings`, and use `settings.LOGGING` instead to pass into `dictConfig`